### ISSUE TITLE
Transitive dependencies have high/critical security vulnerabilities

### ISSUE THREAD

**Issue by mattjohnsonpint** [GoogleCloudPlatform/functions-framework-dotnet#260]
Add this package to a blank project and run `dotnet list package --vulnerable --include-transitive`

```
Project `MyApp` has the following vulnerable packages
   [net6.0]: 
   Transitive Package               Resolved   Severity   Advisory URL                                     
   > Newtonsoft.Json                9.0.1      High       https://github.com/advisories/GHSA-5crp-9r3c-p9vr
   > System.Text.Encodings.Web      4.5.0      Critical   https://github.com/advisories/GHSA-ghhp-997w-qr28
```

**Comment by jskeet** [comment]
Will sort this out on Monday, thanks.

**Comment by jskeet** [comment]
I think these are both actually via CloudNative.CloudEvents.AspNetCore. I'll update that and re-release, then update the dependency here to that new version.

**Comment by jskeet** [comment]
@mattjohnsonpint: Please could you see whether https://github.com/cloudevents/sdk-csharp/issues/239 and https://github.com/cloudevents/sdk-csharp/pull/240 look like the right approach to you?

**Comment by mattjohnsonpint** [comment]
Yes, that seems right to me.

**Comment by mattjohnsonpint** [comment]
This is fixed with version 1.1.0.  Thanks!
